src/renderer/dx/CustomTextLayout.cpp

```diff
@@ -1100,16 +1318,16 @@ void CustomTextLayout::_SplitCurrentRun(const UINT32 splitPosition)
         // In this case, that's 1 corresponding with the ñ.
         const auto mapOffset = _glyphClusters.at(backHalf.textStart);
 
-        // The front half's glyph start index (position in _glyphIndicies)
+        // The front half's glyph start index (position in _glyphIndices)
         // stays the same.
 
-        // The front half's glyph count (items in _glyphIndicies to consume)
+        // The front half's glyph count (items in _glyphIndices to consume)
         // is the offset value as that's now one past the end of the front half.
         // (and count is end index + 1)
         frontHalf.glyphCount = mapOffset;
 
         // The back half starts at the index that's one past the end of the front
-        backHalf.glyphStart = mapOffset;
+        backHalf.glyphStart += mapOffset;

```

Comment:
use chromium math?

Reply:
we have to have a reason to say these things, right? like, glyphStart is a uint32, it can get up to 4 billion before it overflows. are we _worried_ about overflowing here?

Reply:
_4 billion_

Reply:
this means that a single row of text has 4 billion glyphs in it

the storage for those glyphs and the text that originated them takes at least 12GB

Reply:
I think I'm just going to ignore it for now given how tiny the possibility is and that it would make the rest of this harder to read.